pkg/resources/management.cattle.io/v3/globalrolebinding/validator.go

```diff
@@ -220,3 +223,21 @@ func (a *admitter) validateGlobalRole(globalRole *v3.GlobalRole, fieldPath *fiel
 	}
 	return nil
 }
+
+func (a *admitter) validateNamespacedRules(request *admission.Request, rules map[string][]v1.PolicyRule) []error {
```

Comment:
nit: Wondering if that should be shared with the other `validateNamespacedRules` functions. I understand that they're not exactly identical since for GRB you don't need to verify that there's at least one verb.